deploy/ocm-agent-custom-resource/10-ocm-agent-cr.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: OcmAgent
+metadata:
+  name: ocmagent
+  namespace: openshift-ocm-agent-operator
+spec:
+  ocmBaseUrl: ${OCM_BASE_URL}
+  replicas: 1
+  services:
+  - service_log
```

Comment:
to match the name of the service, I think it should be `service_logs`

Reply:
Thank you for the review. To align with OCM api `/api/service_logs` updated the list of spec.services to service_logs